### ISSUE TITLE
solution of 4.3-8 might be wrong

### ISSUE THREAD

**Issue by ACY222** [walkccc/CLRS#532]
Hello, I noticed that in the solution of Problem 4.3-8,  we guess $T(n) \leq cn^2 - cn$ at the beginning,
but get $T(n) \leq cn^2$ rather than $T(n) \leq cn^2 - cn$ in the end. It's incorrect in substituion method.
I haven't found similar issues submitted before, so I submit a new one. Hopefully it's not repeated.

**Comment by jxu** [comment]
Yes I think you're right. It should be for $$1-2c \le -c \iff c \ge 1$$. Then the last two lines are $$cn^2 + (1-2c) n \le cn^2 - cn$$.
